### DISCUSSION TITLE
大佬，能加个番茄钟功能吗

### DISCUSSION THREAD

**Discussion by Johnsonsgithubaccount** [Planshit/Tai#124] [Ideas]
或者用某个软件/分类时间过长了就提醒去干点别的事情
（摸鱼摸太久了想提醒自己该干活了）
（或者是干活太久了提醒自己该摸摸鱼了）

**Comment by Johnsonsgithubaccount** [comment]
刚看了下大佬的其他东西，和ProjectEye合并使用好像就可以了

**Reply by Johnsonsgithubaccount** [reply]
（但是还是希望有摸鱼提醒功能）

**Comment by noberumotto** [comment]
这个功能应该不会加入，会使Tai太臃肿了

**Comment by ocean-sudo** [comment]
> 或者用某个软件/分类时间过长了就提醒去干点别的事情 （摸鱼摸太久了想提醒自己该干活了） （或者是干活太久了提醒自己该摸摸鱼了）

现在 win11 自带一个番茄钟，你可以用，还挺好用的。
![image](https://user-images.githubusercontent.com/51983215/217537568-c9a54197-2732-40f0-89ea-d9c1fd240549.png)

**Reply by Johnsonsgithubaccount** [reply]
可以，刚更新了系统才发现新增了这个功能，感谢
